lib/reverse_sentence.rb

```diff
@@ -1,6 +1,56 @@
 # A method to reverse the words in a sentence, in place.
```

Comment:
Just a note on the comprehension questions  Merge Sort is O(n Log n) not O(log n)

Reply:
lol that's a typo